packages/jest-expect/__typetests__/jest-expect.test.ts

```diff
@@ -5,40 +5,41 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import {
-  expectAssignable,
-  expectError,
-  expectNotAssignable,
-  expectType,
-} from 'tsd-lite';
+import {describe, expect, test} from 'tstyche';
 import {jestExpect} from '@jest/expect';
-import {expect} from 'expect';
-import type {Plugin} from 'pretty-format';
-
-expectType<void>(jestExpect({}).toMatchSnapshot());
-
-expectError(() => {
-  expect({}).toMatchSnapshot();
-});
-
-expectType<void>(jestExpect.addSnapshotSerializer({} as Plugin));
-
-expectError(() => {
-  expect.addSnapshotSerializer();
-});
-
-expectAssignable<typeof expect>(jestExpect);
-expectNotAssignable<typeof jestExpect>(expect);
+import {expect as _expect} from 'expect';
 
 declare module 'expect' {
   interface Matchers<R, T> {
     toTypedEqual(expected: T): void;
   }
 }
 
-expectType<void>(jestExpect(100).toTypedEqual(100));
-expectType<void>(jestExpect(101).not.toTypedEqual(101));
+describe('JestExpect', () => {
+  test('defines the `.toMatchSnapshot()` matcher', () => {
+    expect(jestExpect(null)).type.toHaveProperty('toMatchSnapshot');
+
+    expect(_expect(null)).type.not.toHaveProperty('toMatchSnapshot');
```

Comment:
TSTyche can check if property exists. That’s way better than just `expectError`.